pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/json/JsonIndexType.java

```diff
@@ -95,7 +95,8 @@ public JsonIndexCreator createIndexCreator(IndexCreationContext context, JsonInd
       throws IOException {
     Preconditions.checkState(context.getFieldSpec().isSingleValueField(),
         "Json index is currently only supported on single-value columns");
-    Preconditions.checkState(context.getFieldSpec().getDataType().getStoredType() == FieldSpec.DataType.STRING,
+    Preconditions.checkState(context.getFieldSpec().getDataType().getStoredType() == FieldSpec.DataType.STRING
```

Comment:
(minor) Store `context.getFieldSpec().getDataType().getStoredType()` in a local variable, same for other repeated access

Reply:
^^